CMakeLists.txt

```diff
@@ -18,18 +18,24 @@ list(APPEND CMAKE_MODULE_PATH ${ICUB_MODULE_PATH}
                               ${ICUBCONTRIB_MODULE_PATH}
                               ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-find_package(OpenCV REQUIRED)
-
-set(USE_SIFT_GPU FALSE CACHE BOOL "Select false to run without GPU, or true for GPU acceleration")
-if(USE_SIFT_GPU)
-    find_package(SIFTGPU REQUIRED)
-endif()
+#set(USE_SIFT_GPU FALSE CACHE BOOL "Select false to run without GPU, or true for GPU acceleration")
+#if(USE_SIFT_GPU)
+#    find_package(SIFTGPU REQUIRED)
+#endif()
 
 include(ICUBcontribHelpers)
 include(ICUBcontribOptions)
 icubcontrib_set_default_prefix()
 
+## locate OpenCV libraries
+find_package(OpenCV REQUIRED)
+include_directories(${OpenCV_INCLUDE_DIR})
+
+set(OpenCV_LIBRARIES "opencv_features2d;opencv_imgproc;opencv_core;opencv_cudastereo;opencv_ximgproc;opencv_xfeatures2d")
```

Comment:
Somehow this `set` has appeared again: it was gone at a certain point 🤔 

Anyway, also `include_directories` can be omitted since the upcoming directives `target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES})` will take care of that as well.

Reply:
I've solved this personally with the latest commit 👍